org-formation/100-shared-dns/_tasks.yaml

```diff
@@ -1,24 +1,33 @@
 Parameters:
   <<: !Include '../_parameters.yaml'
 
-# The dns zone for this certificate was created manually, and a
-# certificate for the zone is created in the sageit account in sceptre:
-# ./sceptre/sageit/config/prod/sageit-org-acm-cert.yaml
-#
-# Create a second sageit certificate in admincentral for the finops lambdas
-SageItCertificate:
+# Zone for finops lambdas in admincentral
+SageITFinopsZone:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.6.8/templates/R53-hostedzone.yaml
+  StackName: !Sub '${resourcePrefix}-sageit-finops-zone'
+  StackDescription: Create a shared hosted zone for application domains
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: !Ref SageITAccount
+  Parameters:
+    DnsDomainName: "finops.sageit.org"
+
+# Wildcard certificate for '*.finops.sageit.org'
+SageITFinopsCertificate:
   Type: update-stacks
   Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.6.8/templates/acm-certificate.yaml
-  StackName: !Sub '${resourcePrefix}-sageit-cert'
+  StackName: !Sub '${resourcePrefix}-sageit-finops-cert'
```

Comment:
we need to be very careful here.  a rename will make org-formation delete an existing stack.  We don't want to delete the existing stack because there are dependencies on it.

The better process is to add new resources then carefully deprecate existing resources.

Reply:
fortunately the only dependency on this is the mips-api lambda, but since it's managed earlier than this stack (in `050-costs`) you're right we need to wait until after we've migrated the lambda to the new service before we can remove deprecated cert

Reply:
as a separate matter, i'm wondering if would make sense to move setup of shared-dns to before setup of costs?

Reply:
yeah, I regret the ordering of a lambda in `050-costs` and the dns and certificates in `100-shared-dns`, but I'm not sure which would be better to move